code-of-conduct.md

```diff
@@ -1,23 +1,129 @@
+
+Avoid using out-of-character (OOC) information to unfairly influence in-character (IC) decisions or outcomes. Maintain a clear distinction between your character's knowledge and your own.
+Respect the established lore, timeline, and environment of Woodcrest. Refrain from introducing elements that disrupt the coherence and consistency of the shared world.
+
+
+### Collaborate and Respect Events
+
+Events are a cornerstone of the Woodcrest community, requiring significant time, effort, and coordination to plan and execute. They contribute to the richness and vibrancy of our shared roleplay experience, and it is essential that all residents support and respect these events to ensure their success.
+
+When attending or participating in an event, familiarize yourself with the guidelines and expectations set by the organizers. Follow their instructions and adhere to any specific rules or themes that have been established for the event.
+
+Do not intentionally disrupt or derail scheduled events without prior agreement from the event organizers. Respect the time and effort they have invested in creating an immersive and enjoyable experience for all attendees.
+
+If an unexpected situation arises during an event that would realistically call for a change in focus or direction, proceed with the roleplay in a manner that does not significantly undermine the event's intended purpose or disrupt the experience for other participants.
+
+If you have ideas for substantial changes, additions, or prolonged interruptions to an event, discuss them with the event organizers in advance through out-of-character (OOC) communication. Collaborate with them to find ways to incorporate your ideas seamlessly and in a manner that enhances rather than detracts from the event.
+
+Avoid using OOC influence or information to manipulate events or the actions of other characters. Respect the autonomy and agency of all participants, and allow the roleplay to unfold naturally based on in-character motivations and decisions.
+
+Event organizers have the final say in how to proceed with any unexpected interruptions or changes during their events. Respect their decisions and work with them to maintain a cohesive and enjoyable experience for everyone involved.
+
+### Uphold Community Standards
+
+Woodcrest welcomes a diverse range of avatars, including humans, anthromorphic characters, and ferals. However, we expect all residents to adhere to community standards of decency and respect.
+
+Avatars should be appropriately attired and refrain from overtly sexual or explicit displays in public spaces. Clothing and accessories should be tasteful and non-offensive.
+
+When portraying characters of different ages, especially minors, ensure that your behavior, interactions, and storylines are age-appropriate and do not engage in or depict any form of abuse, exploitation, or sexual content involving underage characters.
```

Comment:
We don't allow minors? This really should state: all characters must be and present over the age of 18. For the latter half we could cover it by saying "don't do anything that breaks LL ToS"

Reply:
Agree, would rather have explicitly termed examples of what is not permitted, including citations to Linden ToS.